frontend/src/concepts/pipelines/context/useManageElyraSecret.ts

```diff
@@ -53,20 +49,13 @@ const useManageElyraSecret = (
         const usingOldUrl = !secretValue.metadata[ELYRA_SECRET_DATA_ENDPOINT].endsWith('/view/');
         if (usingOldDataType || usingOldUrl) {
           // Secret is out of date, update it
-          replaceSecret(
-            generateElyraSecret(
-              dataConnection.data,
-              dataConnection.metadata.name,
-              dataConnection.metadata.namespace,
-              routePath,
-            ),
-          );
+          replaceSecret(generateElyraSecret(externalStorage, dataConnection, routePath));
         }
       } catch (e) {
         // do nothing
       }
     }
-  }, [fullLoadedState, routePath, elyraSecret, dataConnection]);
+  }, [fullLoadedState, routePath, elyraSecret, dataConnection, cr, notification]);
```

Comment:
What's happening here? I'm surprised the linter didn't yell at you -- you never use `notification`? Also, shouldn't you use this in the other hook?

Reply:
Hmm, interesting that I didn't get a lint error here. It's some leftover code when I was trying to figure out the best way to show the errors, I will clean it up.